CHANGELOG

```diff
@@ -47,7 +51,7 @@ Changes
       -Removed DeprecationWarnings
       -Ran against the MySQLdb unit tests to check for bugs
       -Added support for client_flag, charset, sql_mode, read_default_file,
-       use_unicode, cursorclass, init_command, and connect_timeout. 
+       use_unicode, cursorclass, init_command, and connect_timeout.
```

Comment:
this is the only non-related syntax change that it's a real PITA for me to avoid as my editor strips off trailing whitespace.